src/platform/intel/cavs/lib/pm_runtime.c

```diff
@@ -335,6 +383,10 @@ void platform_pm_runtime_get(enum pm_runtime_context context, uint32_t index,
 	case CORE_MEMORY_POW:
 		cavs_pm_runtime_core_en_memory(index);
 		break;
+	case PM_RUNTIME_DSP:
+		index ? cavs_pm_runtime_dis_pg_dsp_slave(index) :
```

Comment:
Please don't replace if/else with ternary operator for execution, it should be used only if you really want return value from one of statements.

Reply:
Fixed in #2057.